Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/NodeTypes/NewsShort.html

```diff
@@ -1,46 +1,60 @@
 {namespace neos=TYPO3\Neos\ViewHelpers}
 {namespace m=TYPO3\Media\ViewHelpers}
 
-<div class="article" itemscope itemtype="http://schema.org/Article">
-	<span itemprop="url">
-		<neos:link.node node="{node}">
+<f:layout name="withCategoryMenu"/>
```

Comment:
Why did you change the NewsShort.html template so much? Looks really weird, I'd revert it.